### DISCUSSION TITLE
Wanted Mac version of SlingPlayer 1.0.10.272 install.

### DISCUSSION THREAD

**Discussion by anonyada** [GerryDazoo/Slinger#528] [General]
Does anyone have access to a specific Slingplayer install .DMG for a specific Mac version of SlingPlayer 1.0.10.272?
Any help, or point in the right direction would be greatly appreciated.  I've looked all over for it.

**Comment by Mikoshi-nyudo** [comment]
> Does anyone have access to a specific Slingplayer install .DMG for a specific Mac version of SlingPlayer 1.0.10.272? Any help, or point in the right direction would be greatly appreciated. I've looked all over for it.

You can find all the files you need for both Mac and PC under my post on HiFi remote, linked here;
http://www.hifi-remote.com/forums/viewtopic.php?p=145528#145528

**Reply by anonyada** [reply]
Thank you so much for the quick reply!  I had already found all of the .DMG's you kindly provided links to. My issue is I'm running
Mac High Sierra and cannot get the older 32 bit version to run (1.0.10.dmg).  From what I understanding in reading this forum, is that in order to get a ProHD to program and rescan ATSC US signals, you need a specific Mac version of SlingPlayer 1.0.10.272.
The thread I'm referring to is yours I believe: 
https://github.com/GerryDazoo/Slinger/discussions/314

The non-phone home versions for windows will scan, but only NTSC.
I have successfully updated the firmware, implemented with Slinger on RPi, ( I also run 350's and I'm not a python virgin), but wanted to get this ATSC ProHD 300-100 unit working to for local digital.

Any ATSC ProHD experience with the 32 bit version, and how to get it running?

**Reply by Mikoshi-nyudo** [reply]
@RandomMan719 did that on-the-ground work and review. He is best to comment. I'm in Australia and not USA.

There's no reason that High Sierra won't run that 32-bit version of SlingPlayer. So you have some other issue going on there.

**Comment by anonyada** [comment]
Problem:  I wanted to run the older version of Slingplayer for MAC(1.0.10.dmg) on my High Sierra install.
You could install it...but Apples "Gatekeeper" on the later versions of MacOS (after Snow Leopard) would prevent you.  Apple themselves, obviously black-listed the older application,
even though it had a valid cert.  Asshats.
Therefore:
For anyone fighting this issue "Later versions of Apple OS will not let you run the older .DMG images after install IE:  "Move it to Trash or Cancel..." Slingplayer.app versions......here you go:

From the CLI....sudo -i to root.

As root:1.  root# ls -al@
This will show you all quarantined installed applications. Note: my Slingplayer.app was quarantined by Apple!
2. To remove the quarantine:
3. Note: (you need to be sitting in the Applications directory and it needs to be done recursively)
	As root:   root# xattr -cr Slingplayer.app
Reboot and test.
